walkthroughs/week-4-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -76,6 +84,18 @@ public void noOptionsForTooLongOfARequest() {
     Collection<TimeRange> expected = Arrays.asList();
 
     Assert.assertEquals(expected, actual);
+
+    // Case when there are other events scheduled
+    Collection<Event> testCaseWithSchedules = new ArrayList<Event>(Arrays.asList(Events.events));
```

Comment:
Additional test should be in separate test case.
This ensures that the test is testing one thing at a time.